common/helpers.go

```diff
@@ -277,3 +277,16 @@ func k6ThrowUnsafe(ctx context.Context, format string, a ...interface{}) {
 // tb := testBrowser(t, withContext(ctx))
 // cancel()
 var k6ThrowOnce sync.Once
+
+// TrimQuotes removes surrounding single or double quotes from s.
+// We're not using strings.Trim() to avoid trimming unbalanced values,
+// e.g. `"'arg` shouldn't change.
+// Source: https://stackoverflow.com/a/48451906
+func TrimQuotes(s string) string {
+	if len(s) >= 2 {
+		if c := s[len(s)-1]; s[0] == c && (c == '"' || c == '\'') {
+			return s[1 : len(s)-1]
+		}
+	}
+	return s
+}
```

Comment:
Sorry, I disagree. Helpers/utils are helpful regardless of how many times they're used. In the same way we wouldn't copy `strings.Trim()` inline if we used it instead, we shouldn't move this either.

Helpers are useful by simply existing as long as they have a single use, since it makes them discoverable for anyone that might find them useful later, instead of them re-implementing something similar inline as well.

Besides, this is a lower level of abstraction than `setFlagsFromArgs` or `Parse`. This implementation doesn't concern either of them.